docs/explorer.rst

```diff
@@ -12,4 +12,395 @@ Api Reference
 Usage and examples
 ------------------
 
-ToDo
\ No newline at end of file
+Initializing an Explorer object
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We establish a connection to Sumo by initializing an Explorer object.
+This object will handle authentication and can be used to retrieve cases and case data.
+
+.. code-block::
+
+    from fmu.sumo.explorer import Explorer 
+
+    sumo = Explorer()
+
+
+Authentication
+^^^^^^^^^^^^^^^
+If you have not used the `Explorer` before and no access token is found in your system, a login form will open in your web browser.
+It is also possible to provide the `Explorer` with an exisiting token to use for authentication, in this case you will not be prompted to login.
+
+.. code-block:: 
+
+    from fmu.sumo.explorer import Explorer 
+
+    USER_TOKEN="123456789"
+    sumo = Explorer(token=USER_TOKEN)
+
+This assumes the `Explorer` is being used within a system which handles authentication and queries Sumo on a users behalf.
+
+Finding a case
+^^^^^^^^^^^^^^
+The `Explorer` has a property called `cases` which represents all cases you have access to in Sumo:
+
+.. code-block::
+
+    from fmu.sumo.explorer import Explorer 
+
+    sumo = Explorer() 
+
+    cases = sumo.cases 
+
+The `cases` property is a `CaseCollection` object and acts as a list of cases.
+We can use the `filter` method to apply filters to the case collection which will return a new filtered `CaseCollection` instance:
+
+.. code-block::
+
+    from fmu.sumo.explorer import Explorer 
+
+    sumo = Explorer() 
+
+    cases = sumo.cases
+    cases = cases.filter(user="peesv")
+
+In this example we're getting all the cases belonging to user `peesv`.
+
+The resulting `CaseCollection` is iterable:
+
+.. code-block::
+
+    from fmu.sumo.explorer import Explorer 
+
+    sumo = Explorer() 
+
+    cases = sumo.cases
+    cases = cases.filter(user="peesv")
+
+    for case in cases:
+        print(case.uuid)
+        print(case.name)
+        print(case.status)
+
+We can use the filter method to filter on the following properties:
+- uuid
+- name
+- status
+- user
+- asset
+- field
```

Comment:
I guess this is meant to render as a list, but readthedocs doesn't want to?
![image](https://user-images.githubusercontent.com/89020325/222172113-c1c22afe-4c7b-4d59-a09f-26ab4646d32d.png)